impl/client/src/main/javascript/web/pentaho/visual/action/MessageExecution.js

```diff
@@ -0,0 +1,68 @@
+/*!
+ * Copyright 2010 - 2019 Hitachi Vantara. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define([
+    "module",
+    "./InteractionExecution",
+    "./ModelChangedError"
+], function(module, InteractionExecution, ModelChangedError) {
+
+    /**
+     * @name MessageExecution
+     * @memberOf pentaho.visual.action
+     * @class
+     * @extends pentaho.action.impl.Target.ActionExecution
+     *
+     * @classDesc The execution class for an
+     * [Update]{@link pentaho.visual.action.Update} action in a
+     * [Model]{@link pentaho.visual.Model}.
+     *
```

Comment:
This add been created to address the issue on this [comment](https://github.com/pentaho/pentaho-platform-plugin-common-ui/pull/1572#discussion_r918099497).
It has since been removed, as the Message action now extends `Base`